src/material/bottom-sheet/bottom-sheet.md

```diff
@@ -45,7 +45,7 @@ import {MAT_BOTTOM_SHEET_DATA} from '@angular/material/bottom-sheet';
   template: 'passed in {{ data.names }}',
 })
 export class HobbitSheet {
-  constructor(@Inject(MAT_BOTTOM_SHEET_DATA) public data: any) { }
+  constructor(@Inject(MAT_BOTTOM_SHEET_DATA) public data: { names: string[] }) { }
```

Comment:
nit: usually we don't have spaces inside of object literals (e.g. `{names: string[]}`).

Reply:
@crisbeto I am curious, is this a coding style that you follow?

Reply:
It's how we've always been writing them. It might be coming from the Google JS style guide.